koji-scm-lib/src/main/java/org/fakekoji/xmlrpc/server/xmlrpcresponse/ArchiveList.java

```diff
@@ -0,0 +1,48 @@
+package org.fakekoji.xmlrpc.server.xmlrpcresponse;
+
+import hudson.plugins.scm.koji.Constants;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.fakekoji.xmlrpc.server.xmlrpcrequestparams.XmlRpcRequestUtils.toMaps;
+
+public class ArchiveList implements XmlRpcResponse<List<String>> {
+
+    private final List<String> archives;
+
+    public ArchiveList(List<String> archives) {
+        this.archives = archives;
+    }
+
+    public ArchiveList(Object object) {
+        archives = parseArchiveMaps(toMaps(object));
+    }
+
+    @Override
+    public Object toObject() {
+        return new Object[]{};
```

Comment:
surprising, now I'm even more confused what this `toObject` means

Reply:
Normally, it shoud return array of maps (as array of objects), but it is not implemented in fake-koji so I return empty array